python/lsst/daf/butler/core/dataUnit.py

```diff
@@ -235,47 +251,6 @@ def table(self):
         """
         return getattr(self, '_table', None)
 
-
-class DataUnitRegion:
```

Comment:
There is small bunch of comments that still refer to `DataUnitRegion` class (they don't show up on this diff), they need cleanup too.